ui/src/components/Map/MarkerPopup.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <LPopup>
+    <h3>Node: <a :href="`${baseNodeUrl}${node.id}`" target="_blank">{{ node.label }}</a></h3>
+
+    <span class="larger-icon"><FeatherIcon :icon="Location" /></span>
+    {{ toFixed(node.assetRecord.latitude, 6) }}, {{ toFixed(node.assetRecord.longitude, 6) }}
+    <br />
+    <a :href="getTopologyLink(node)">View in Topology Map</a>
+    <br />
+    <br />
+    <div class="flex">
+      <div>IP Address:</div>
+      <div>{{ ipAddressForNode(node) || 'N/A' }}</div>
```

Comment:
Refactored to pass in `ipAddress` and create it in the parent component

For `MarkerClusterPopupContent`, I refactored a bit. Want to keep `ipInterfaceMap` in one place, in the parent. So still pass `ipListForNode` into `MarkerClusterPopupContent` since it needs to derive IP addresses for all its children.